transports/tls/CHANGELOG.md

```diff
@@ -2,6 +2,7 @@
 
 - Migrate to `{In,Out}boundConnectionUpgrade` traits.
   See [PR 4695](https://github.com/libp2p/rust-libp2p/pull/4695).
+- Bump `rcgen` to `0.11.0`.
```

Comment:
```suggestion
```

We normally don't mention dependency bumps in the changelog because it would clutter them too much.

Reply:
Ah, sorry. Thanks for letting me know. I'll keep it in mind in the future.